addons/website_blog/controllers/main.py

```diff
@@ -148,6 +148,13 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
     ], type='http', auth="public", website=True, sitemap=True)
     def blog(self, blog=None, tag=None, page=1, **opt):
         Blog = request.env['blog.blog']
+
+        if not request.params and tag not in request.params and tag:
```

Comment:
Hi,

I don't see any check on the request ype.

Reminder:
/blog/tag/1 -> **get** ok  ! make it retrocompatible for multi tags to redirect to canonical url with first tag only.
/blog/tag/1,3 -> **Post** ok

Disallow multi tag in **get**, to avoid stupid bot indexation

Reply:
Else, first check seems very good !